Bat.bat

```diff
@@ -112,22 +113,27 @@ REM Saving parameters to variables...
 	Set _9=%~9
 )
 
+
 REM Acting as per the Passed parameters...
 Set _Valid=F
-
-if /i "%_1%" == "Update" (Call :Update && Set _Valid=T)
-if /i "%_1%" == "ilist" (Call :Installed_List && Set _Valid=T)
-if /i "%_1%" == "list" (Call :List && Set _Valid=T)
-if /i "%_1%" == "search" (Call :Search && Set _Valid=T)
-if /i "%_1%" == "install" (Call :Install && Set _Valid=T)
-if /i "%_1%" == "uninstall" (Call :Uninstall && Set _Valid=T)
-if /i "%_1%" == "detail" (Call :Details && Set _Valid=T)
-if /i "%_1%" == "reset" (If /I "%_2%" == "all" (Call :ResetAll) ELSE (Call :Reset) && Set _Valid=T)
+for %%P in (update ilist list search install uninstall detail reset move) do (
+	IF /i "%_1%" == "%%P" (set _Valid=True)

```

Comment:
As you ware working inside FOR loop, You need to use "!" instead of "%" to expand variables properly. Otherwise, CMD can show some errors.
@GroophyLifefor